oauth2/oauth2-resource-server/src/test/java/org/springframework/security/oauth2/introspection/TestOAuth2TokenIntrospectorResponses.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2002-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.oauth2.introspection;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.net.URL;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.springframework.security.oauth2.introspection.OAuth2IntrospectionClaimNames.ACTIVE;
+import static org.springframework.security.oauth2.introspection.OAuth2IntrospectionClaimNames.AUDIENCE;
+import static org.springframework.security.oauth2.introspection.OAuth2IntrospectionClaimNames.CLIENT_ID;
+import static org.springframework.security.oauth2.introspection.OAuth2IntrospectionClaimNames.EXPIRES_AT;
+import static org.springframework.security.oauth2.introspection.OAuth2IntrospectionClaimNames.ISSUER;
+import static org.springframework.security.oauth2.introspection.OAuth2IntrospectionClaimNames.NOT_BEFORE;
+import static org.springframework.security.oauth2.introspection.OAuth2IntrospectionClaimNames.SCOPE;
+import static org.springframework.security.oauth2.introspection.OAuth2IntrospectionClaimNames.SUBJECT;
+import static org.springframework.security.oauth2.introspection.OAuth2IntrospectionClaimNames.USERNAME;
+
+public class TestOAuth2TokenIntrospectorResponses {
```

Comment:
Since introspection does not have a domain object, I'm struggling with the right name for this class. `TestMaps` isn't right.

I'd like the name to stay focused on the domain, if possible, so that this class can be used for both the imperative and reactive introspection tests. Thoughts?

Reply:
Note that since this comment, I've renamed this to `TestOAuth2TokenIntrospectionClientResponses`. The question still stands, though, as this rename was simply to align with the change to `OAuth2TokenIntrospectionClient`.